contacts/templates/contacts/contact.html

```diff
@@ -0,0 +1,18 @@
+{% load staticfiles %}
+
+<link rel="stylesheet" type="text/css" href="{% static 'contacts/style.css' %}" />
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>My Contacts</title>
+</head>
+<body>
+<h1>My Contacts</h1>
+
+    {% for name in my_contacts %}
```

Comment:
the items in `my_contacts`list are not just `name` but `contact`, change the var here!

Reply:
done